lahja/base.py

```diff
@@ -366,6 +383,13 @@ def serve(cls, config: ConnectionConfig) -> AsyncContextManager["BaseEndpoint"]:
         """
         ...
 
+    @abstractmethod
+    async def connect_to_endpoints(self, *endpoints: ConnectionConfig) -> None:
```

Comment:
Unrelated to this PR but should we just remove the non-variadic `connect_to_endpoint`. Since the variadic one can be called with `1..n` connection configs it seems unnecessary to keep an extra API for the single parameter use case, no?

Reply:
https://github.com/ethereum/lahja/pull/137